src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,32 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
```

Comment:
Would `humanYears` rather than `number` be a better parameter name here? That way you could skip the intermediate variable on line 3.